toqito/state_props/is_separable.py

```diff
@@ -7,7 +7,7 @@
 from toqito.matrix_props import is_positive_semidefinite
 from toqito.state_props import is_ppt, in_separable_ball
 from toqito.state_props.has_symmetric_extension import has_symmetric_extension
-from toqito.state_metrics import trace_norm
+from toqito.matrix_props.trace_norm import trace_norm
```

Comment:
Similar comment here:

```suggestion
from toqito.matrix_props import trace_norm
```